recipes/intbitset/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "intbitset" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d4ceba6c252223a2e60072b8e42b867004eb31972098e0d16924cf0d27aed387
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - six
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - intbitset
+
+about:
+  home: http://github.com/inveniosoftware/intbitset/
+  license: GNU Lesser General Public v3 or later (LGPLv3+)
+  license_family: LGPL
+  license_file: lgpl-3.0.txt
```

Comment:
Looking at the upstream code, you could put `LICENSE` here and it should work, no need for bundling it with the recipe.

Reply:
I was not sure exactly what you meant here, but I've removed the license file and replaced the section with 'LICENSE'